internal/upgrade/upgrade.go

```diff
@@ -61,7 +62,12 @@ func Run(cfg *TestConfig) {
 			replicas, err := state.GetFramework().GetExpectedControlPlaneReplicas(state.GetContext(), state.GetCluster().Name, state.GetCluster().GetNamespace())
 			Expect(err).NotTo(HaveOccurred())
 
-			Eventually(wait.Consistent(common.CheckControlPlaneNodesReady(state.GetContext(), wcClient, int(replicas)), 12, 5*time.Second)).
+			Eventually(
+				wait.ConsistentWaitCondition(
```

Comment:
I'm probably late with this comment, but isn't this the same as the gomega func [Consistently](https://onsi.github.io/gomega/#consistently)?

Reply:
When the `clustertest` framework was originally built we didn't want to tie it to gomega where we didn't have to so that it was more flexible.